llvm/include/llvm/IR/IntrinsicsAMDGPU.td

```diff
@@ -1991,7 +1991,9 @@ class AMDGPURawBufferLoadLDS : Intrinsic <
                               //        gfx12+: bits [0-2] = th, bits [3-4] = scope,
                               //                bit 6 = swz
                               //           all: volatile op (bit 31, stripped at lowering)
-  [IntrWillReturn, NoCapture<ArgIndex<1>>, ImmArg<ArgIndex<2>>, ImmArg<ArgIndex<5>>,
+  [IntrWillReturn, IntrArgMemOnly,
```

Comment:
This isn't correct.

`raw.buffer.load.lds` accesses the memory pointed to by the `<4 x i32>` resource.

Reply:
Which already have it

Reply:
Yes, that's one of the problems. All users should migrate to the pointer buffer intrinsics, it's not worth doing anything with the legacy vector descriptors

Reply:
... We need to migrate LLPC over, for one thing

And actually announce the deprecation